packages/expo-updates/e2e/setup/project.ts

```diff
@@ -48,7 +48,7 @@ function getExpoDependencyChunks({
       ? [['expo-dev-menu-interface'], ['expo-dev-menu'], ['expo-dev-launcher'], ['expo-dev-client']]
       : []),
     ...(includeTV
-      ? [['expo-av', 'expo-blur', 'expo-image', 'expo-linear-gradient', 'expo-localization']]
+      ? [['expo-av', 'expo-blur', 'expo-image', 'expo-linear-gradient', 'expo-localization', 'expo-crypto', 'expo-network']]
```

Comment:
Out of scope from this PR, but I think it would be a good idea to decouple the `expo-updates` E2E tests from all of these other packages, and keep the tests solely focused on updates and modules that directly interact with updates such as the dev client.

Reply:
That's a good idea... maybe take all of the infrastructure in `project.ts` that sets up a new project with arbitrary packages, and put it into an expotool that could be used anywhere for E2E tests.